playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -641,9 +641,10 @@ EDXAPP_DATADOG_PROFILING_ENABLE: "{{EDXAPP_DATADOG_ENABLE and COMMON_ENABLE_DATA
 # spans in the webapp and faceting metrics by service.
 # https://docs.datadoghq.com/tracing/guide/inferred-service-opt-in/?tab=python
 EDXAPP_DATADOG_INFERRED_SERVICES_ENABLE: true
-# ddtrace 2.7.9 contains a fix for a pymongo incompatibility.
-# (Same fix is present in 2.8.2 on the 2.8.x release line.)
-EDXAPP_DDTRACE_PIP_SPEC: 'ddtrace>=2.7.9'
+
+# We are seeing issues with celery processing tasks with later versions of ddtrace,
+# so this pin is in place until we understand what's going on.
```

Comment:
It would be great if you could link to the follow-up ticket, whatever that may.

Reply:
Followup is https://github.com/edx/edx-arch-experiments/issues/1060